lib/src/summarize.dart

```diff
@@ -4,12 +4,51 @@
 
 library dart_pad.summarize;
 
+import '../services/dartservices.dart';
 /// Instances of this class take string input of dart code and output a text 
 /// description ofthe code's size, packages, and other useful information
 class Summarizer {
   
-  String summarize(String input) {
-    return "foo";
+  String summarize(SourceRequest input, AnalysisResults result) {
+    if (input == null || result == null) {
+      return "Summarizer has broken!.";
+    }
+    String summary = '<pre><code><b>Summary (Under Development)</b><p/>';
+    summary += '${_linesCode(input)} lines of code used. <p/>';
+    bool hasErrors = result.issues.any((issue) => issue.kind == 'error');
+    bool hasWarnings = result.issues.any((issue) => issue.kind == 'warning');
+    if (hasErrors) summary += 'Errors are present.<p/>';
```

Comment:
Some ideas to make this more playful:

```
Oops! 3 analysis errors. This snippet is pretty broken!

To err is human; this snippet has 73 of them!
```

Let's take the summary text as a WIP. Can you file an issue to improve it to something closer to a hand-written summary? The purpose of this PR is integrating it into the sharing workflow, so I don't want to distract from that in this PR.